tools/makefiles/recursive.mk

```diff
@@ -69,7 +69,16 @@ endif
 
 clean: git-clean
 
-$(RECURSIVE_TARGETS): .clean-error-log $(MAKE_DIRS) .print-error-log
+# Special target that only runs at root level (not recursive)
+.PHONY: ai-context-files
+ai-context-files:
+	@echo "Building AI context files..."
+	@python tools/build_ai_context_files.py
+	@python tools/build_git_collector_files.py 2>/dev/null || echo "Note: git-collector not available, skipping external docs"
+	@echo "AI context files generated in ai_context/generated/"
```

Comment:
this target uses paths relative to the root of the repo. this file, recursive.mk, is included in a Makefiles through the repo.

this target should be added tot he Makefile at the root - not this file